src/Microsoft.Health.Fhir.Api.UnitTests/Features/Security/ResourceActionHandlerTests.cs

```diff
@@ -32,8 +31,8 @@ public ResourceActionHandlerTests()
         [InlineData(true)]
         public async void GivenAReadResourceRequest_WhenAuthorizationHandlerHandles_ThenTheAuthorizationHandlerAppropriateStatus(bool authorized)
         {
-            _authorizationPolicy.HasPermission(Arg.Any<ClaimsPrincipal>(), ResourceAction.Read).ReturnsForAnyArgs(authorized);
```

Comment:
I think this was inadvertently deleted.

Reply:
Oops I thought I ran the tests before submitting it but apparently I didn't...